### PR TITLE
Redesign mvp

### PR DESCRIPTION
**Pull request by NicholasKissel**

### CODE REVIEW

**Review by NathanFlurry — CHANGES_REQUESTED** [review]
Need to update game engine graphics, fix remaining links

File: src/pages/index.jsx [hunk 1]
```diff
@@ -287,6 +304,41 @@ let caseStudies = [
...
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/html5/webrtc',
+    screenshot: imgWeb,
+  },
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/rust/cli-websocket',
+    screenshot: imgCli,
+  },
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/c/coredumpo',
```
Comment (MasterPtato) [inline src/pages/index.jsx:337]: typo, should be https://github.com/rivet-gg/examples/tree/main/c/coredumpohttps://github.com/rivet-gg/examples/tree/main/c/coredump

File: src/pages/index.jsx [hunk 2]
```diff
@@ -602,6 +669,109 @@ function Demo() {
...
+    },
+    Unreal: {
+      image: imgStepsGodot,
+    },
+    HTML5: {
+      image: imgStepsGodot,
+    },
+    Custom: {
+      image: imgStepsGodot,
+    },
```
Comment (MasterPtato) [inline src/pages/index.jsx:696]: Are these supposed to all be the Godot image?

**Review by MasterPtato — CHANGES_REQUESTED** [review]

File: src/components/GlowBox.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import grainTexture from 'src/images/effects/grain.png'; 
+
+const creamColor = '#f0e5d6';
```
Comment (MasterPtato) [inline src/components/GlowBox.jsx:4]: Nit: SCREAMING_CAMEL_CASE

**Review by MasterPtato — CHANGES_REQUESTED** [review]

**Review by NathanFlurry — APPROVED** [review]

**Pull request merged**